### ISSUE TITLE
Minutes and agenda items

### ISSUE THREAD

**Issue by lwjohnst86** [steno-aarhus/research#7]
Please only use this for minutes and agenda items. I've copied them over from the Google Doc.

**Comment by lwjohnst86** [comment]
# 2020-10-28

## Agenda

1. Introduction to topic (why have this, etc) (~5 min)
2. Brainstorming 1: Individually write out as many ideas for things to cover or include in the "common docs"/"manual"/"standard operating procedures" as possible (maybe about 5-10 min) and put onto stickies/post-it notes
    1. Afterwards, take all the stickies and organize them and discuss them. (maybe about 15 min)
3. Brainstorming 2: In small groups (pairs or threes), based on previously discussed ideas, expand on current ideas or come up with other ideas. Again, write on stickies. (maybe about 15 min)
    2. Afterwards, take stickies, organize them, and discuss (~15 min)
4. Introduction to some basic things we need/should have for implementing these common docs. (~10 min). Includes: 
    3. Relatively easy for anyone to contribute to; 
    4. Easily share-able (both internally and externally to potential collaborators);
    5. Able to write and interact with docs in either Danish or English;
    6. Easily searchable/navigable/shows list of all docs available;
    7. (other ideas?)
5. Brainstorming 3: In new small groups, think about as many ideas as possible for how to go about implementing this. (~10 min)
    8. Gather stickies and discuss ideas all together (~10 min)
6. (This depends on the outcome of Brainstorming 3 above) Brainstorming 4: Individually, write out as many names as possible on stickies for what to call this (if we e.g. make a website or something for this). (~5 min)
    9. Gather stickies and discuss (~5 min).
7. Post-brainstorming: 
    10. Make a committee responsible for this?
    11. Create and assign roles for this committee?

## Minutes

*   See [notes-from-session](https://docs.google.com/document/d/10RKCGFnCwUzHhpVbw7nrm6rTNVBmouKv_yF0O4KTFeI/edit#) for what we brainstormed
*   Action items:
    *   Start using Google Docs for immediate term to developing common docs
    *   Adam will showcase using current website and if it's feasible to use as our “entry point”
    *   Luke to showcase Git at Steno Thursday?
    *   First common doc: Onboarding for newcomers
    *   Add others to Google Folder (**DONE)**
    *   Put pictures up online (**DONE)**
    *   Link to other Google Docs?
    *   Luke: Send when to meet next, in a month? **(DONE)**
    *   _Tinne:_ Digitize discussion points and put into a [brainstorming notes](https://docs.google.com/document/d/10RKCGFnCwUzHhpVbw7nrm6rTNVBmouKv_yF0O4KTFeI/edit) file **(DONE)**

**Comment by lwjohnst86** [comment]
# 2020-12-02

## Agenda

1. Update on action item: Creating onboarding material for newcomers.
2. **(LUKE) **Discussion: Establish a few principles for decision making and creation of the docs:
    - _“All are welcome to contribute”_: There should be no or little barrier to contributing to the documents and their content, aside from barriers due to technology and skill in its use. If someone has the motivation and desire to contribute, they should be able to fairly easily.
    - _“Inclusion and participation in community through active contribution”_: Individuals who want to participate should actively contribute through not just attending meetings, BUT ALSO in creating, modifying, organizing, editing, and/or developing the content. Those who actively contribute should be included in the community and encouraged to participate. This will help prevent having people in the community who request content but don’t actively contribute, so that anything we make is based on our own motivation rather than from being told to.
    - _“Content is dictated by direct and active contribution”_: What gets created or developed should be done by whoever contributes to creating or building it. This:
        - Prevents people internal or external to the community from asking or requesting for things to be added and implicitly suggesting someone in the community to do it
        - Prevents any individual or group of individuals from being placed as responsible for work that is not paid for and ensures that responsibility and work is distributed as much as possible.
        - Lowers the barrier to what gets created from “requiring permission” to “if you want it, make it”.
    - _“Important or significant decisions are based on community consensus”_: Decisions should not or need not be blocked by requiring permissions for anything external to the community. This will ensure things move forward as continuously and unrestricted as possible. Important or significant decisions could include things like approving and finalizing a document or deciding on major infrastructural changes (e.g. platform to use for creating and managing the content). 
    - Ultimately, this common doc project is largely voluntary and shouldn’t become something that takes up too much time or mental energy and that doesn’t get inhibited by too much bureaucracy.
3. **(SINE?) **Summary of our needs and what has been found about each:
    - _In general, RM expects research needs to be met through AU connection._
    - Server for data space
        - Data storing: servers (also Linux servers) can be provided from RM IT. It will be possible to access the server from a Linux computer as long as you have a RM user.
    - Intranet for SDCA employees
        - Current SDCA intranet site has some of the info that also needs to be on our potential research intranet site but otherwise doesn’t seem suited for the research purpose.
        - Other examples: E.G. Department of Molecular Medicine, but they have their own IT department and servers, which gives them more flexibility. Or e.g. Præhospitalet who work with external partners, but requires funding.
        - We could have our own IT department but will require our own funding.
    - Website for research purposes, common docs (suitable for public), research projects, etc.
        - We can create our own website (nothing is stopping us from doing that from a Region level according to Morten in IT). In regard to RM requiring a specific design:  RM is mostly interested in places where there is patient interaction and as this webpage needs to work on a daily basis, RM will probably be happy if at least the logo is there.
        - GitHub fits a lot of the needs and it should be possible to link to it as long as there is no patient information. (Shouldn't be a problem as there is not data storing only code). Within GitHub you can make your own website and it is hosted by GitHub.
        - An Idea could be to add a subpage to the SDCA webpage – need to ask Marie (and give her examples for inspiration). Here it would also be possible to link to a webpage we make ourselves.
4. Which potential locations to create and put these materials and which meets our needs most?
    - Quick presentation/walkthrough: **Adam** will show off how to edit the website (as a potential place we can link the common docs or for other purposes) (_Either for intranet or for website)_
    - Discussion: Other options? (Luke will show his proposed solution based on GitHub during the next meeting.)
5. Discussion: Next steps and assigning responsibilities and roles?
    - Idea: After developing these things, writing a paper describing our experience, how to use the resources for others, the challenges facing us, and how it helped us work better (this will likely take a year or two before we write this). Example of what this might look like is: [https://www.nature.com/articles/s41559-017-0160](https://www.nature.com/articles/s41559-017-0160)
    - Who wants to lead which section?

## Minutes

1. Minute taker: Alisa
2. Luke went through the four principles, that can be seen as the code of conduct.
3. _Discussion on potential obstacles. _
    *   _github: Luke will make a video on how to use _
    *   _what documents would be included in “common docs” _
    *   _truly voluntary and motivated aspect versus given task_
    *   _version control, package maintainer_
    *   _Collective experience of the group gathered (learning from past experience and knowledge sharing)_
    *   _Involving administrative staff in making the organization structure (Tinne suggested to invite a staff representative to this working group)_
4. _Sine went through the three topics discussed with Morten from RM IT_
    1. Server for data space: Luke will take a closer look at this
    2. Intranet for SDCA employees: Contact and see Præshospitalet (private vendor-needs funding), MOMA solution (KI, not regional with own IT dpt)  is not an option
    3. Website for research purposes, common docs (suitable for public), research projects, etc.: it is possible to link out from the SDCA webpage as long as the logos are there, possibility to control access to github/webpage with password/log-on if docs not for sharing
    4. Adam showed how to edit content on the webpage (only if you have this enabled). 
        - Discussion on having 2 systems-the webpage and github-need to maintain both. Consensus that the webpage could be substantially improved.
        - Website-flashy news to the outside, links to github-the boring dictionary
5. Next steps and assigning responsibilities and roles:
    1. Next time, Luke will show his proposed solution based on GitHub during the next meeting.
    2. Discuss the linked paper at next meeting
    3. Tinne and Adam will have a look at DST document (uploaded to github afterwards)
    4. Daniel is working on newcomers doc, working as external (employed from Sweden)
    5. After the AI course and JC organized by Adam, 2 data projects: 2 teams: one working “oldschool” and 1 working on github.

**Comment by lwjohnst86** [comment]
# 2021-01-06

## Agenda

1. Update on action items from the previous meeting.
2. **(LUKE)** Brief walkthrough on how to use GitHub for writing docs.

## Minutes

*   Slowly try out GitHub for making documentations
*   Action items:
    *   Adam: Data application documentation
    *   Luke: Update to contributing with video tutorial
        *   Making issues with things to do
    *   Sine: Legal about issues
    *   Alisa: Tutorial or FAQ for using GitHub (for her own project)
*   Inspiration for onboarding material from the Department of Public Health, Research Unit for Epidemiology (Ask Daniel for link)

**Comment by lwjohnst86** [comment]
# 2021-02-03


## Details



*   **Time**: 10:00
*   **Location**: Zoom meeting


## Agenda



1. Update on action items from the previous meeting.
    1. **Luke:** Has created two videos with a tutorial on contributing to the docs. See [https://steno-aarhus.github.io/research/CONTRIBUTING.html#navigating-the-github-project](https://steno-aarhus.github.io/research/CONTRIBUTING.html#navigating-the-github-project) 
        1. Try this method out and discuss how it goes a few months from now.
2. Discussion: Thoughts on writing a paper describing our experience, how to use the resources for others, the challenges facing us, and how it helped us work better (this will likely take a year or two before we write this). Example: [https://www.nature.com/articles/s41559-017-0160](https://www.nature.com/articles/s41559-017-0160)
3. Discussion: Move meeting agenda and minutes to Trello? Or keep this Google Doc?


## Minutes



*   Updates and random:
    *   Would be nice to have some support sessions (would be better in person). Maybe for now, have online sessions to help each other out, like “a support group”?
        *   When? Luke will send out invites to informal session days (?) 
    *   DigitalOcean has good documentation on using GitHub etc.
    *   Maybe have other videos, showing workflow with others?
*   Paper discussion:
    *   Would need to be targeted or something rather than just about this website
    *   Maybe, around data and GDPR? Building a research structure/infrastructure? How to build an open science structure in this context?
    *   Maybe pick a few tools and write about/document how to move and use them?
    *   What’s the aim of the (potential) paper? Something to compare that it has improved our workflow or that it works.
    *   We’ll in the meantime think about it and what to compare, what to improve, and what data to record to present.
    *   Idea: Describe how it was done in small steps so others could replicate and use. Or that clinicians can be convinced and also start using it.
    *   Idea: Start documenting steps and learning in a blog as the first step to writing the paper.
        *   Maybe involve others and ask them to write something? Like research staff.
*   Trello vs Google Doc agenda and minutes:
    *   For this group, move to GitHub project instead, so it’s all right there.
*   Action items:
    *   **Luke** send out doodle on when to meet for co-working session and learning for using this workflow
    *   **Luke** make video on using Markdown
        *   And other videos on workflow and collaborating (e.g. commenting on Pull Requests, making Issues)
    *   **Luke** will add all our brainstorm items to issues on project
    *   **Luke** will add blog capability to project
        *   Maybe add our minutes to the meetings as blog posts.
        *   Luke will make a video of adding blog content?
    *   **Everyone**: Create GitHub account and send Luke your username
    *   **Everyone**, after Luke has added tasks/brainstorm items, indicate which one you want to be responsible for.

**Comment by lwjohnst86** [comment]
# 2021-02-03

## Agenda

1. Update on action items: 
    - Complete #21?
    - Look through "Content to add" issues and select which you want to lead.

## Minutes

**Comment by lwjohnst86** [comment]
# 2021-03-24

## Agenda

- Progress updates.
- Someone else responsible for organizing meetings and sending out calendar invites?
- What should be on landing page? (On this page: https://steno-aarhus.github.io/research/)

**Comment by danielibsen** [comment]
> # 2021-03-24
> ## Agenda
> * Progress updates.
>   
>   * Main organization list: #25
> * Someone else responsible for organizing meetings and sending out calendar invites?
> * What should be on landing page? (On this page: https://steno-aarhus.github.io/research/)
> 
> ## Minutes
> * Main organizing list:
>   
>   * Add "how to setup studies" (@sineknorr) and "medical lab doc" (Ulla and Ole).
> * Update from @danielibsen: Making table with a bunch of information in it. Will convert to Google Sheet and give anyone access.
> * Luke (with help from Alisa) will continue organizing meetings.
> * Ideas for landing page. Put in #30
> * Others:
>   
>   * It'd be great to have a common calendar, but will need more organizing across SDCA to implement.

Hi all. Here is the link to a table of the topics that we plan to include in the onboarding material. Please go to the link, look at the table and add any topics you think would be relevant to include. 

https://docs.google.com/document/d/1uszx7HE9ivaz5hNvdfvUCzPSNAExX3VCHrUFjq1mae8/edit?usp=sharing 

@tinnelaurberg and I will talk about the topics with Maria and convert the table to a GoogleSheet.

**Comment by lwjohnst86** [comment]
# 2021-04-21

## Agenda

- Progress updates.

## Minutes

**Comment by lwjohnst86** [comment]
# 2021-05-18

## Agenda

- Progress updates

## Minutes

**Comment by lwjohnst86** [comment]
don't need this Issue anymore
